### PR TITLE
Fix/master/ro locale names

### PR DESCRIPTION
**Pull request by vicusbass**
This adds Romanian locale to Faker, person provider (gender based)

### CODE REVIEW

**Comment by fcurella** [comment]
Thank you @vicusbass :)

Could you add the source of your data as comments above the list of names?

**Comment by vicusbass** [comment]
Done, added comments.
Basically, scrapping Wikipedia and checking the list manually is one source. Myself, another source, I added some well-known Romanian last names to the list.

File: faker/providers/person/ro_RO/__init__.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        'Pompiliu', 'Radu', 'Rafael', 'Rareș', 'Raul', 'Răducu', 'Răzvan', 'Relu', 'Remus', 'Robert', 'Romeo',
+        'Romulus',
+        'Sabin', 'Sandu', 'Sandu', 'Sava', 'Sebastian', 'Sergiu', 'Sever', 'Severin', 'Silvian', 'Silviu', 'Simi',
+        'Simion',
+        'Sinică', 'Sorin', 'Stan', 'Stancu', 'Stelian', 'Șerban', 'Ștefan', 'Teodor', 'Teofil', 'Teohari', 'Theodor',
+        'Tiberiu', 'Timotei', 'Titus', 'Todor', 'Toma', 'Traian', 'Tudor', 'Valentin', 'Valeriu', 'Valter', 'Vasile',
+        'Vasilică', 'Veniamin', 'Vicențiu', 'Victor', 'Vincențiu', 'Viorel', 'Visarion', 'Vlad', 'Vladimir', 'Vlaicu',
+        'Voicu', 'Zamfir', 'Zeno')
+
+    # sources: https://ro.wikipedia.org/wiki/Lista_celor_mai_uzuale_nume_de_familie#Rom%C3%A2nia and myself
```
Comment (fcurella) [inline faker/providers/person/ro_RO/__init__.py]: Could you please remove yourself from the list? One of the reasons we want sources is to delegate the editorial responsibility away from us. By modifying the list we are making an editorial choice and we can't claim that the list comes from there anymore.

Comment (vicusbass) [reply faker/providers/person/ro_RO/__init__.py]: Makes sense, removed the extra names.

**Comment by fcurella** [comment]
Thank you! ✨

**Pull request merged**